bazel_tools/rules_nodejs_npm_cli_path.patch

```diff
@@ -1,30 +0,0 @@
-diff --git a/internal/node/node_repositories.bzl b/internal/node/node_repositories.bzl
-index 087c9ada..6eef8186 100644
---- a/internal/node/node_repositories.bzl
```

Comment:
Is this unrelated or does your change make this unnecessary because the directory structures look the same on unix and Windows?

Reply:
Oh, sorry, didn't mean to include that one here, yet. I've removed this change. I'll look into this one separately. This currently still needed to make e.g. `bazel run //language-support/ts/daml-json-types:npm_package.pack` work.